rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -258,6 +241,14 @@ class MenuController extends ControllerBase implements ApplicationContextAware{
             return redirect(controller: 'menu', action: 'home')
         }
         def startpage = params.page?: grailsApplication.config.rundeck.gui.startpage ?: 'jobs'
+
+        // Trigger project home access event
+        auditEventsService.eventBuilder()
```

Comment:
I think this trigger should not be put here. I would suggest maybe instead implementing a grails Interceptor that matches certain url paths to observe as project access points. 

(This specific controller action should probably be removed/deprecated. I think it is still used, but is something I would like to remove at some point.)

Also, repeated access to the same project should not trigger another event, correct? Perhaps you will need to have a value stored in the user session indicating if a project has been accessed, and use that to determine if the event needs to fire.